esmvaltool_sample_data/loader.py

```diff
@@ -0,0 +1,89 @@
+from pathlib import Path
+
+import cf_units
+import iris
+
+base_dir = Path(__file__).parent
+
+
+def strip_attributes(cube):
+    """Remove attributes that cause issues with merging and concatenation."""
+    for attr in ['creation_date', 'tracking_id', 'history']:
+        if attr in cube.attributes:
+            cube.attributes.pop(attr)
+
+
+def simplify_time(cube):
+    coord = cube.coord('time')
+    coord.convert_units(
+        cf_units.Unit('days since 1850-1-1 00:00:00',
+                      calendar=coord.units.calendar))
+
+
+def load_cubes_from_input_dirs(input_dirs):
+    """Loads all *.nc files from each input dir into a cube."""
+    for input_dir in input_dirs:
+        files = input_dir.glob('*.nc')
+        cubes = iris.load(str(file) for file in files)
+        for cube in cubes:
+            strip_attributes(cube)
+            simplify_time(cube)
+
+        cubes = cubes.concatenate()
+        cube = cubes[0]
+
+        yield cube
+
+
+def load_timeseries_cubes():
+    """
+    Data: ta / Amon / historical / r1i1p1f1, any grid, 1850 - onwards.
+    All dimensions reduced to a few steps except for the time dimension
+    Some other variable / ocean, probably a different frequency,
+       similar number of timesteps, other dimensions reduced.
```

Comment:
For now I just included monthly and daily `ta` (air temperature) in #4.

Reply:
It would be nice if this function could support some filtering of the cubes, to select cubes with certain attributes (e.g. monthly or daily data)

Reply:
Yeah, I added daily/monthly filtering for now.